api/envoy/extensions/filters/network/http_connection_manager/v4alpha/http_connection_manager.proto

```diff
@@ -478,16 +478,33 @@ message HttpConnectionManager {
   // Should paths be normalized according to RFC 3986 before any processing of
   // requests by HTTP filters or routing? This affects the upstream *:path* header
   // as well. For paths that fail this check, Envoy will respond with 400 to
-  // paths that are malformed. This defaults to false currently but will default
-  // true in the future. When not specified, this value may be overridden by the
-  // runtime variable
+  // paths that are malformed. This defaults to true.
+  // This by default affects the upstream *:path* header as well.
```

Comment:
Actually reading this I was really confused about what was meant by affecting the upstream path header.  Can we maybe rephrase to contract normalizing path for in-Envoy code, vs normalizing the path *sent* uptream?